src/diagnostics/default_diagnostics.jl

```diff
@@ -92,7 +92,7 @@ daily_max(short_names; output_writer = HDF5Writer()) =
 Return a list of `ScheduledDiagnostics` that compute the daily min for the given variables.
 """
 daily_mins(short_names...; output_writer = HDF5Writer()) =
-    common_diagnostics(24 * 60 * 60, min, output_writer, short_names...)
+    common_diagnostics(24 * 60 * 60, nothing, output_writer, short_names...)
```

Comment:
I think this should be `min`. Why did you change to `nothing`?

Reply:
Oh, this is a typo, thanks!